records_mover/records/schema/schema/known_representation.py

```diff
@@ -60,7 +60,7 @@ def from_dataframe(df: 'DataFrame',
     def from_db_driver(driver: 'DBDriver',
                        schema_name: str,
                        table_name: str) -> 'RecordsSchemaSqlKnownRepresentation':
-        type = f"sql/{driver.db.dialect.name}"
+        type = f"sql/{driver.db_engine.dialect.name}"
```

Comment:
this is kinda nice, as a reader i can look at this code and reason, yeah an engine has a dialect with a name that we are referencing vs thinking wtf is this db object i hope its an engine lol

Reply:
haha, def. The use of the term db throughout was just hella confusing on my first read throughs.

Reply:
Done!